docs/authentication/saml/account-linking.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Account Linking
+description: Learn how Clerk handles account linking during SAML SSO.
+---
+
+# Account Linking
+
+Account Linking is a process used by Clerk to ensure a smooth sign-in and sign-up experience using both SAML SSO and other methods (e.g. username/password). By using the "email address" as the common identifier, Clerk aims to merge accounts whenever possible. This mechanism is triggered when an SAML provider returns an email address that matches an existing account, assuming a single owner for each email address.
```

Comment:
```suggestion
Account Linking is a process that Clerk uses to ensure a smooth sign-in and sign-up experience using SAML SSO and other methods (e.g. username/password). By using the email address as the common identifier, Clerk automatically attempts to link accounts whenever possible. Account linking triggers when a SAML provider returns an email address that matches an existing account, assuming a single owner for each email address.
```

Really good intro! Just some cleanup edits here. I added the word "automatically". Is that technically accurate? I thought it would make it more explicit that Clerk is coded to do this, rather than any people managing it